### ISSUE TITLE
session.subscribe() is not working on XCode 11.3.1

### ISSUE THREAD

**Issue by sarangborude** [aciidb0mb3r/SwiftMQTT#52]
Publish is working fine. 
I am making the subscribe call once the connection happens, but there is no response to the subscribe call, no error or sucess

**Comment by ream88** [comment]
Yeah, the `completion` callback is never called, however my app is able to receive messages.

**Comment by dermdaly** [comment]
I've seen this issue too.  It looks the the subscribe request is sent out with a message id of 1, but the response comes back with a message id of 0.  This means the relevant callback won't be called.  I wonder if it is a remote side issue as the message ids are wrong.

**Comment by tyagishi** [comment]
PR#45(https://github.com/aciidb0mb3r/SwiftMQTT/pull/45) solve this issue.
i.e. relevant callback will be called.

I checked on Xcode11.6.
